docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -151,6 +151,35 @@ The deprecated `universal_discovery_widget_module.default_location_id` setting h
 
 - Deprecated `SubtreeQuery` class has been removed. In v3.0 it was replaced by `\EzSystems\EzPlatformAdminUi\QueryType\SubtreeQueryType`.
 
+#### View matching
+
+When matching views using custom services, the services must be now tagged with `ezplatform.view.matcher`.
+The matching must be configured in the following way:
+
+``` yaml
+content_view:
+    full:
+        folder:
+            template: folder.html.twig
+            match:
+                '@App\Matcher\MyMatcher': ~
+```
+
+#### SiteAccess matching
+
+When matching SiteAcceses using custom services, the SiteAccess must be now tagged with `ezplatform.siteaccess.matcher`.
+The matching must be configured in the following way:
+
+``` yaml
+siteaccess:
+    list: [site]
+    groups:
+        site_group: [site]
+    default_siteaccess: site
+    match:
+        '@App\Matcher\MySiteaccessMatcher': 1
```

Comment:
I think we can remove siteaccess matcher example as hasn't change. The only change is a requirement to tag a service.